sdk/python/feast/diff/FcoDiff.py

```diff
@@ -16,23 +16,28 @@
 from feast.protos.feast.core.RequestFeatureView_pb2 import (
     RequestFeatureView as RequestFeatureViewProto,
 )
+from feast.registry import FeastObjectType, Registry
+from feast.repo_contents import RepoContents
 
-FcoProto = TypeVar(
-    "FcoProto",
-    EntityProto,
-    FeatureViewProto,
-    FeatureServiceProto,
-    OnDemandFeatureViewProto,
-    RequestFeatureViewProto,
-)
+FEAST_OBJECT_TYPE_TO_STR = {
+    FeastObjectType.ENTITY: "entity",
+    FeastObjectType.FEATURE_VIEW: "feature view",
+    FeastObjectType.ON_DEMAND_FEATURE_VIEW: "on demand feature view",
+    FeastObjectType.REQUEST_FEATURE_VIEW: "request feature view",
+    FeastObjectType.FEATURE_SERVICE: "feature service",
+}
+
+FEAST_OBJECT_TYPES = FEAST_OBJECT_TYPE_TO_STR.keys()
```

Comment:
nit: alternative more robust way probably is to do [e.value for e in FeastObjectType]